app/src/test/kotlin/nl/jvandis/teambalance/AbstractIntegrationTest.kt

```diff
@@ -0,0 +1,106 @@
+package nl.jvandis.teambalance
+
+import nl.jvandis.jooq.support.migrations.LiquibaseSupport
+import org.jooq.DSLContext
+import org.jooq.SQLDialect
+import org.jooq.impl.DSL.using
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.BeforeEach
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.util.TestPropertyValues
+import org.springframework.context.ApplicationContextInitializer
+import org.springframework.context.ConfigurableApplicationContext
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.transaction.annotation.Transactional
+import org.testcontainers.containers.PostgreSQLContainer
+import org.testcontainers.junit.jupiter.Container
+import org.testcontainers.junit.jupiter.Testcontainers
+import java.sql.Connection
+import java.sql.DriverManager
+
+@Testcontainers
+@Transactional("transactionManager")
+@SpringBootTest
+@ActiveProfiles("integration-test")
+@ContextConfiguration(initializers = [AbstractIntegrationTest.Initializer::class])
+@AutoConfigureMockMvc
+class AbstractIntegrationTest {
+
```

Comment:
I'm trying to do a little bit of integration testing as well. For a solo / hobby project it's nice to at least have some form of non-manual quality assurance in place.